cypress/e2e/SmartPicker.spec.js

```diff
@@ -0,0 +1,83 @@
+import { initUserAndFiles, randUser } from '../utils/index.js'
+
+const currentUser = randUser()
+
+const fileName = 'empty.md'
+
+describe('Smart picker', () => {
+	before(() => {
+		initUserAndFiles(currentUser, 'empty.md')
+	})
+
+	beforeEach(() => {
+		cy.login(currentUser)
+		cy.visit('/apps/files')
+	})
+
+	it('Type and see the smart picker', () => {
+		cy.isolateTest({
+			sourceFile: fileName,
+		})
+		cy.openFile(fileName, { force: true })
+
+		cy.getContent()
+			.type('/')
+
+		cy.get('.tippy-box .suggestion-list').children().should(($children) => {
+			const entries = $children.find('.suggestion-list__item').map((i, el) => el.innerText).get()
+			expect(entries.length).to.be.greaterThan(0)
+			expect('To-Do list').to.be.oneOf(entries)
+			expect('Table').to.be.oneOf(entries)
```

Comment:
Is this still expected after the recent reordering that put default actions on top again?

Reply:
Should be fine for the test as there is only very limited default pickers. Will check though.